manuscript/draft.Rmd

```diff
@@ -640,42 +633,37 @@ the habitats and species being surveyed [@sauer1994]. Accounting for differences
 in observers reduced the average error in our point estimates and also improved
 the coverage of the confidence intervals. In addition, controlling for observer
 effects resulted in changes in which models performed best, most notably
-reducing the relative performance of the naive model's point estimates. This
-suggests that, prior to accounting for observer effects, the naive model
-performed well because it was capable of accommodating rapid shifts in estimated
-richness introduced by changes in the observer. These kinds of rapid changes
-were difficult for the other single-site models to accommodate, and so the
-average performance of the ARIMA and average models improved once this source of
-observation error was addressed. This demonstrates that modeling observation
-error can be important for properly estimating and reducing uncertainty in
-forecasts and can also lead to changes in the best methods for forecasting [Box
-1]. Another key aspect of an ideal observation model is imperfect detection. In
-this study, we did not address differences in detection probability across
-species and sites [@boulinier1998] since there is no clear way to address this
-issue using NA Breeding Bird Survey data without making strong assumptions about
-the data [i.e., assuming there is no biological variation in stops along a
-route; @white2010], but this would be a valuable addition to future forecasting
-models.
+improving most models' point estimates relative to the naive baseline [Box 1].
```

Comment:
This Box 1 citation looks like it is associated with "improving most models' point estimates relative to the naive baseline", which doesn't make sense to me.